src/anbox/dbus/stub/application_manager.cpp

```diff
@@ -40,7 +40,7 @@ ApplicationManager::ApplicationManager(const BusPtr& bus)
 ApplicationManager::~ApplicationManager() {}
 
 void ApplicationManager::update_properties() {
-  bool ready = false;
+  int ready = false;
```

Comment:
Changing to `int` is fine but we should drop the use of `false` then and use `0`

Reply:
Makes sense. Thanks for the review. Fixed it.